### PR TITLE
Autoclose Embeded file feed

### PR DESCRIPTION
**Pull request by 20michaelz**

### CODE REVIEW

File: packages/filefeeds/src/filefeeds.ts [hunk 1]
```diff
@@ -158,20 +158,20 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
         try {
           const resumeToken = window.localStorage.getItem(this.#resumeTokenKey)
           if (resumeToken) {
-            mergedParams.resumeToken = resumeToken
+            mergedParams.sessionToken = resumeToken
```
Comment (behnam-oneschema) [inline packages/filefeeds/src/filefeeds.ts:161]: I think we changed this and you need to revert your change here.

Comment (behnam-oneschema) [reply packages/filefeeds/src/filefeeds.ts:161]: Basically, we decided to—similar to the Importer SDK—the FF SDK sends the `resumeToken` value directly to the init endpoint and let that endpoint decide which ones of `sessionToken` and `resumeToken` to use.

File: packages/filefeeds/src/filefeeds.ts [hunk 2]
```diff
@@ -158,20 +158,20 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
...
+      if (OneSchemaFileFeedsClass.#iframeIsLoaded) {
         this._initWithRetry(mergedParams)
+      } else if (this.iframe) {
+        this.iframe.onload = () => {
+          OneSchemaFileFeedsClass.#iframeIsLoaded = true
+          this._initWithRetry(mergedParams)
+        }
       }
     }
   }
```
Comment (behnam-oneschema) [inline packages/filefeeds/src/filefeeds.ts:177]: Why did we change this?

File: packages/filefeeds/test/index.ts [hunk 3]
```diff
@@ -30,8 +30,15 @@ const resumeTokenKey = `OneSchemaFileFeeds-session-${userJwt}`
 
 const fileFeeds = oneSchemaFileFeeds({
   parentId: "oneschema-container",
+<<<<<<< HEAD
```
Comment (behnam-oneschema) [inline packages/filefeeds/test/index.ts:33]: These look like merge issues.

File: packages/filefeeds/src/filefeeds.ts [hunk 4]
```diff
@@ -362,6 +362,7 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
             /* local storage is not available, don't sweat it */
           }
         }
+        this.#hide()
```
Comment (behnam-oneschema) [inline packages/filefeeds/src/filefeeds.ts:365]: Thanks!

One question, though: should we just `#hide()` at this point, or `destroy()`?

**Review by behnam-oneschema — APPROVED** [review]

File: packages/filefeeds/src/filefeeds.ts (hunk 4 above)
Comment (20michaelz) [reply packages/filefeeds/src/filefeeds.ts:365]: I think it should prob be destroy but lets have the change to destroy happen in a different pr along w/ the changes to close

**Pull request merged**